web-test-runner.config.js

```diff
@@ -104,9 +104,8 @@ export default {
         ],
 
   // If a test suite takes longer than this, it's almost certainly hanging and
-  // won't finish. 2 minutes is the default. These timeouts can be reduced by
-  // 30 seconds or so when Dropdown is moved out of the repository.
-  testsFinishTimeout: process.env.CI ? 90_000 : 60_000,
+  // won't finish. 2 minutes is the default.
+  testsFinishTimeout: process.env.CI ? 120_000 : 60_000,
```

Comment:
Menu's interactions test suite will be mostly boilerplate but massive, and it takes some time to run. It's not strictly necessary to add this now. But I wanted to reduce the size of my upcoming Menu PR as much as possible.